CMakeLists.txt

```diff
@@ -298,24 +304,48 @@ if(MSVC)
 	endif()
 endif()
 
-install(TARGETS yaml-cpp ${_INSTALL_DESTINATIONS})
-install(
-	DIRECTORY ${header_directory}
-	DESTINATION ${INCLUDE_INSTALL_DIR}
-	FILES_MATCHING PATTERN "*.h"
-)
+set(VERSION ${YAML_CPP_VERSION})
```

Comment:
As far as I understand the `VERSION` variable used only in `write_basic_package_version_file`. So it can be just `write_basic_package_version_file(... VERSION "${YAML_CPP_VERSION}")`.